modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -40,7 +40,7 @@ You can see the configured TLS security profile in the `APIServer` custom resour
 
 [NOTE]
 ====
-The control plane does not support TLS `1.3` and because the `Modern` profile requires TLS `1.3`, it is not supported.
+The control plane does not support TLS `1.3` as the minimum TLS version and because the `Modern` profile requires TLS `1.3`, it is not supported.
```

Comment:
```suggestion
The control plane does not support TLS `1.3` as the minimum TLS version, and because the `Modern` profile requires TLS `1.3`, it is not supported.
```